client/blocks/credit-card-form/test/index.js

```diff
@@ -13,15 +13,19 @@ import React from 'react';
 /**
  * Internal dependencies
  */
-import CreditCardForm from '../';
+import { CreditCardForm } from '../';
 
 jest.mock( 'i18n-calypso', () => ( {
 	localize: x => x,
 } ) );
 
+// Gets rid of warnings such as 'UnhandledPromiseRejectionWarning: Error: No available storage method found.'
```

Comment:
This look like a commit message rather. I think a better comment would be to indicate which dependency causes this warning